src/pages/Loans/LoansOverview/components/LoanModal/BorrowForm.tsx

```diff
@@ -143,6 +146,7 @@ const BorrowForm = ({ asset, variant, position }: BorrowFormProps): JSX.Element
             aria-label={content.fieldLabel}
             {...register(amountFieldName)}
           />
+          <LoanScore score={collateralRation?.toNumber()} />
```

Comment:
Conversion to `number` won't be needed after https://github.com/interlay/interbtc-ui/pull/627 is merged.